### PR TITLE
Cameron wilcox fizzbuzz plus plus

### PR DESCRIPTION
**Pull request by cameronwilcox**
Pull request for Cameron Wilcox, having added to Cory Janes program.

### CODE REVIEW

**Review by Skeegan123 — COMMENTED** [review]
I would recommend splitting your unit tests into their own tests i.e. a test for 3, a test for 5, a test for 7, and a test for 11. I did the same thing originally but the rubric says "unit tests" so there is probably supposed to be multiple. Let me know if that makes sense. Other than that it looks pretty good. I will try running it when you make that change.

**Comment by cameronwilcox** [comment]
@Skeegan123 Made the changes, let me know if I should change anything else.

**Review by Skeegan123 — COMMENTED** [review]
+1 Looks good. I will pull it down and test it

**Review by Skeegan123 — CHANGES_REQUESTED** [review]
A couple things, 1. Could you try changing the run.sh from python to python3 and see if it works on your computer? That way it will work on macs too since from what I've seen most macs need it to be python3 and that usually works on windows as well. And 2. Could you try adding the permissions to test.sh and run.sh then commiting them? If you already did its whatever but for me I had to add the permissions myself.

File: fizzbuzz/python/coryjanes/fizzbuzz.py [hunk 1]
```diff
@@ -1,23 +1,26 @@
...
+	if number % 3 == 0:
+		to_print = to_print + "Fizz"
+	# if number is divisible by 5, display 'Buzz'
+	if number % 5 == 0:
+		to_print = to_print + "Buzz"
+	if number % 7 == 0:
+		to_print = to_print + "Ping"
+	if number % 11 == 0:
+		to_print = to_print + "Pong"
+	if number % 5 != 0 and number % 3 != 0 and number % 7 != 0 and number % 11 != 0:
```
Comment (sleddog) [inline fizzbuzz/python/coryjanes/fizzbuzz.py:21]: could simplify this and check to see: `if to_print == ""`

**Comment by cameronwilcox** [comment]
@Skeegan123 I should have updated the permissions, let me know if it works for you.

**Review by Skeegan123 — APPROVED** [review]
+10 Looks good and runs properly with the correct permisions and simplified if statement

**Pull request merged**